GettingStarted.md

```diff
@@ -1,80 +1,165 @@
 
-Optionally, if you wish to enable spatial mapping in your scene, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
+> **Note**
+> If you enable the **Enable Xbox Controller Support** option, this will download the ProjectSettings Input file and replace your current version in your project and override any existing input settings, the old file will be renamed to "*.Old*".
+> Else, you will have to setup all the MRTK input axis manually.
 
-# 5. Building your project for HoloLens
- 1. Be sure to plug in your HoloLens via usb.
- 2. Navigate to the Build Window: `HoloToolkit -> Build Window`.
- 3. Under `Deploy` be sure to fill out the device Username and Password fields.
- 4. Under `Quick Options` press: `Build SLN, Build APPX, then Install`.
+# 5. Preparing a Scene for Mixed Reality Content
+
+Select the "Apply Mixed Reality Scene Settings" option in the Unity Editor: 
+
+` Mixed Reality Toolkit -> Configure -> Apply Mixed Reality Scene Settings`
+
+![MRTK Editor Scene Settings](/External/ReadMeImages/MixedRealityEditorSceneSettings.png)
+
+Select all the required options for your Scene type:
+
+![MRTK Scene options](/External/ReadMeImages/MixedRealitySceneOptions.png)
+
+Alternatively, you can setup your scene manually as follows:
+
+1. Create a new Scene: `File -> New Scene`
+
+2. Remove the default `Main Camera` and `Directional Light` objects in the scene.
+
+3. Add the `MixedRealityCameraParent.prefab` (found under HoloToolkit/Input/Prefabs). Check the configured options for the Parent and child camera meet your requirements.
+
+4. Add the `Managers.prefab` (found under HoloToolkit/Input/Prefabs/Cursor), which will add a new Managers folder containing the **InputManager** Prefab and the all-important UI **EventSystem** object.
+
+5. Add the `DefaultCursor.prefab` (found under HoloToolkit/Input/Prefabs/Cursor) and add that Object to the **InputManager** Cursor parameter (to avoid it being searched for on scene start) 
+
+> **For Hololens**
+> Optionally, if you wish to enable spatial mapping in your scene for HoloLens, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
+
+# 5 Building your project 
+The MRTK provides you a quick and easy way to generate your Unity project from a custom window, which can be found under:
+
+`Mixed Reality Toolkit -> Build Window`
+
+![MRTK Build Window](/External/ReadMeImages/MixedRealityEditorBuildWindow.png)
+
+This window offers many quick options to be able to:
+
+* Build your UWP C# solution
+* Build your project APPX
+* (HoloLens) Deploy your project to a remote device
```

Comment:
Should we also describe which SDKs, build configs. etc and to target per platform?

Reply:
Only needed for HL, which is why it's included there.  
Will do a second look and see if we also need a "Deploying WMR" section